### PR TITLE
[Corda] Add Corda validation docs

### PR DESCRIPTION
**Pull request by jwavoetacn**
Signed-off-by: jwavoetacn <68590662+jwavoetacn@users.noreply.github.com>

**Changelog**
- Added Corda validation documentation

**To be reviewed by**
@suvajit-sarkar 

**Linked issue**
#883

### CODE REVIEW

**Comment by suvajit-sarkar** [comment]
The table formatting is not correct. You can preview from here :
https://github.com/jwavoetacn/blockchain-automation-framework/blob/feature/883-r3-corda-checks/docs/source/operations/baf_verify.md

File: docs/source/operations/baf_verify.md [hunk 1]
```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
...
+The flow chart shows the R3 Corda process. To verify the steps of deployment, follow the Verification Table 'R', to troubleshoot the general errors.
+
+![](./../_static/corda_flowchart.png)
+
+----
+### R3 Corda Troubleshooting
+### Table 'N'
+
+|Section|Sub-Section|Problem|Possible Cause|Solution|
+|----|---|----------|-------|--------------|--------|
```
Comment (sownak) [inline docs/source/operations/baf_verify.md:256]: Table formatting needs to be corrected

File: docs/source/operations/baf_verify.md [hunk 2]
```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
...
 
+
+---  
+
+
+## R3 Corda Checks
+
+The flow chart shows the R3 Corda process. To verify the steps of deployment, follow the Verification Table 'R', to troubleshoot the general errors.
+
+![](./../_static/corda_flowchart.png)
```
Comment (sownak) [inline docs/source/operations/baf_verify.md:249]: Flowchart should contain registration check(because without that the node will not progress).

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by jagpreetsinghsasan — COMMENTED** [review]
This corda troubleshooting guide is even smaller than that of Quorum (Quorum has just one chart for one TM and Corda has like 5+ charts) 😁

File: docs/source/operations/baf_verify.md (hunk 1 above)
Comment (jwavoetacn) [reply docs/source/operations/baf_verify.md:256]: That is odd, the VS Code preview had no issues, let me check.

Comment (jwavoetacn) [reply docs/source/operations/baf_verify.md]: Fixed

File: docs/source/operations/baf_verify.md (hunk 2 above)
Comment (jwavoetacn) [reply docs/source/operations/baf_verify.md:249]: Added registration check to the flowchart

File: docs/source/operations/baf_verify.md [hunk 3]
```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
...
+
+----
+### R3 Corda Troubleshooting
+### Table 'N'
+
+|Section|Sub-Section|Problem|Possible Cause|Solution|
+|-------|-----------|-------|--------------|--------|
+| R1 | a | **Ansible playbook failed** <br>Playbook execution terminated at <br>**Role:** create/certificates/ambassador <br>**Task:** Copy generated ambassador tls certs to given build location. <br>**Error:** Destination directory, example: /home/[user]/build/corda/doorman/tls, does not exist | Folder to copy tls certs does not exist. | **network.network_services.certificate** value is either misspelled or directory doesn't exist. |
```
Comment (sownak) [inline docs/source/operations/baf_verify.md:257]: If this is open-source Corda, still following items are missing from the flowchart

1. Notary registration failure
2. Notary DB failure
3. Node DB Failure.

Also, the flowchart talks about R1-R5, but in the table here, there is only R1 and R2. What is a user supposed to do when they go to R5?

**Review by sownak — CHANGES_REQUESTED** [review]
Seems this needs a thorough review.

**Comment by sownak** [comment]
@jwavoetacn Can this be completed now?

**Comment by jwavoetacn** [comment]
> @jwavoetacn Can this be completed now?

Making some final adjustments (with help of @suvajit-sarkar) according to your given feedback.

**Pull request closed without merging**